License2Deploy/rolling_deploy.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+
+import logging
+import boto.ec2 as ec2
+import boto.ec2.autoscale as a
+import boto.ec2.elb as e
+from sys import exit, argv
+from time import sleep, time
+
+class RollingDeploy(object):
+
+  def __init__(self, env=None, project=None, buildNum=None, ami_id=None, profile_name='default'):
+    self.env = env
+    self.project = project.replace('-','')
+    self.buildNum = buildNum
+    self.ami_id = ami_id
+    self.region = self.determine_region(self.env)
+    self.profile_name = profile_name
+    self.exit_error_code = 2
+
+  def aws_conn_auto(self):
+    return a.connect_to_region(self.region, profile_name=self.profile_name)
+
+  def aws_conn_ec2(self):
+    return ec2.connect_to_region(self.region, profile_name=self.profile_name)
+
+  def aws_conn_elb(self):
+    return e.connect_to_region(self.region, profile_name=self.profile_name)
+
+  def determine_region(self, env):
+    if env == 'qa':
+      return 'us-west-1'
+    elif env in ['stg', 'prd']:
+      return 'us-east-1'
+    else:
+      return env # This method is somewhat specific to our environment, so I am allowing the option for env to equal an actual region and will return the original value supplied
```

Comment:
I would suggest moving this into a separate file/class and calling it statically. 
My thinking is this will depend on who uses the application.
It should be blank and filled in by users as a configs file